minecraft_servers/dragonpvp/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+    "server_name": "dragonpvp",
+    "nice_name": "DragonPvP Network",
+    "direct_ip": "play.dragonpvp.it",
+    "supported_languages": [
+      "it"
+    ],
+    "social": {
+      "web": "https://www.dragonpvp.site",
+      "web_shop": "https://www.dragonpvp.site/store",
+      "web_support": "https://www.dragonpvp.site/forum",
+      "discord": "https://dsc.gg/dsdragonpvp"
+    },
+    "gamemodes": {
+      "KitPvP": {
+        "name": "KitPvP",
+        "command": "/queue kitpvp",
+        "color": "#0095B0",
+        "versions": "1.8<1.20.1"
+      },
+      "Bedwars": {
+        "name": "Bedwars",
+        "command": "/queue bedwars",
+        "color": "#ffffff"
+      },
+      "uhc": {
+        "name": "UHC",
+        "color": "#537928",
+        "command": "/queue uhc"
+      }
+    },
+    "discord": {
+      "server_id": 884523075503259709,
+      "rename_to_minecraft_name": true

```

Comment:
Please use false, as soon is only allowed for verified partners.

Reply:
done